.github/workflows/lint.yml

```diff
@@ -0,0 +1,36 @@
+name: Lint
```

Comment:
Please use the ones provided by app-tutorial https://github.com/nextcloud/app-tutorial/blob/master/.github/workflows/lint.yml